app/controllers/admin/uphold_reports_controller.rb

```diff
@@ -0,0 +1,30 @@
+require 'csv'
+
+module Admin
+  class UpholdStatusReportsController < AdminController
+    def index
+      @uphold_report = UpholdStatusReport.
+        group('(EXTRACT(YEAR FROM created_at))::integer').
+        group('(EXTRACT(MONTH FROM created_at))::integer').
+        order('2 DESC, 3 DESC').count
+    end
+
+    def show
+      date = DateTime.strptime(params[:id], "%Y-%m")
+      start_date = date.at_beginning_of_month
+      end_date = date.at_end_of_month
+
+      @uphold_report = UpholdStatusReport.where("created_at >= :start AND created_at <= :finish", start: start_date, finish: end_date)
```

Comment:
This shouldn't be using `@`. Prefer renaming from `@uphold_report` to `uphold_status_report`.